.github/workflows/ci.yaml

```
@@ -56,6 +61,22 @@ jobs:
             helm unittest -u "$chart"
           done
 
+  helm:
```

actionlint (1.7.12)

[error] key "helm" is duplicated in "jobs" section. previously defined at line:36,col:3. note that this key is case insensitive

(syntax-check)